composer.json

```diff
@@ -15,17 +15,22 @@
         "php": ">=5.3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.0.15",
+        "phpunit/phpunit": "4.5.*",
         "satooshi/php-coveralls": "dev-master",
-        "codeless/jugglecode": "1.0",
-        "vlucas/phpdotenv": "1.0.6"
+        "codeless/jugglecode": "1.0"
     },
     "autoload": {
         "psr-0": {
             "Bigcommerce": "src/"
+        },
+        "psr-4": {
+            "Bigcommerce\\Test\\": "test/"
         }
     },
     "archive": {
         "exclude": ["bin", "bigcommerce.php"]
```

Comment:
just for point of reference, this directory and file are removed in my branch, will remove here also when this gets merged and pulled in.